warehouse/accounts/models.py

```diff
@@ -111,6 +120,22 @@ class UnverifyReasons(enum.Enum):
     SoftBounce = "soft bounce"
 
 
+class TwoFactor(db.ModelBase):
+    __tablename__ = "two_factor"
+    __table_args__ = (
+        UniqueConstraint("user_id", name="two_factor_user_key"),
+        Index("two_factor_user_id", "user_id"),
```

Comment:
If I remember correctly, this is going to create a duplicate index to the ``UniqueConstraint`` above, just one with and without the unique flag set. We should double check that, but I think we can drop the ``Index`` here on PostgreSQL.